enhancements/image-based-installer/image-based-installer.md

```diff
@@ -0,0 +1,219 @@
+---
+title: image-based-installer
+authors:
+  - "@mresvanis"
+  - "@eranco74"
+reviewers:
+  - "@patrickdillon"
+  - "@romfreiman"
+approvers:
+  - "@zaneb"
+api-approvers:
+  - None
+creation-date: 2024-04-30
+last-updated: 2024-04-30
+tracking-link:
+  - https://issues.redhat.com/browse/MGMT-17600
+see-also:
+  - "/enhancements/agent-installer/agent-based-installer.md"
+replaces: N/A
+superseded-by: N/A
+---
+
+# Image-based Installer
+
+## Summary
+
+The image-based installer is an installation method for on-premise single-node
+OpenShift (SNO) clusters, that will use a bootable, installer image running on
+the hosts that are to become SNO clusters (much like what is implemented by the
+functionality of the [Agent-based Installer](/enhancements/agent-installer-agent-based-installer.md))
```

Comment:
I think it would be helpful to explicitly state what distinguishes this image-based install method from the agent-based installer. Why can't we use Agent to do this?

Reply:
I believe the intent here was to state that the installer already implements the functionality to generate an ISO with embedded ignition config.
I'll add the reasoning for image based installation with regards to what differentiates it from Agent based installation.